### ISSUE TITLE
Building with JDK9 in "JAVA_HOME"

### ISSUE THREAD

**Issue by TheMrMilchmann** [TheMrMilchmann/Osmerion#18]
Currently attempting to build the project fails because the Gradle Kotlin scripts cannot compile. This is caused by a bug in Kotlin 1.1.3 that causes compilations to fail on JDK9. Unfortunately this is not in my hand to solve as I cannot set the version of the compiler the Gradle Kotlin DSL uses without building the entire project myself. However, as this bug has already been solved in the Kotlin 1.1.4 snapshots, it should only be a matter of time until this issue is solved.

TL;DR
- [x] Wait for the Kotlin 1.1.4 release
- [x] Wait for the Gradle Kotlin DSL to upgrade to Kotlin 1.1.4
- [ ] Update Gradle distro

**Comment by TheMrMilchmann** [comment]
Just a little update:
Kotlin 1.1.4 has been released and the Kotlin DSL has been updated but unfortunately it still fails to compile scripts on JDK9.
